control-plane/p4RuntimeSymbolTable.cpp

```diff
@@ -0,0 +1,400 @@
+/*
```

Comment:
I will assume that definitions in this file are just moved from the other source.

Reply:
Yes, I only added a new function call.